### DISCUSSION TITLE
Passing seed value to OpenAI APIto get mostly consistent response

### DISCUSSION THREAD

**Discussion by Omkar-Shetkar** [spring-projects/spring-ai#190] [Q&A] [answered]
According to https://cookbook.openai.com/examples/deterministic_outputs_with_the_seed_parameter, OpenAI API providing provision to pass seed value along with prompt. With current implementation of Spring AI, `ChatCompletionRequest` has seed field. However, probably we need to provide a mechanism to pass it through `OpenAiClient`. 
Is it something deferred because seed is in beta or any other reason?
Thanks.

**Comment by tzolov** [comment, marked as answer]
Hey @Omkar-Shetkar , thank you for bringing this up.
I believe the new options support: https://github.com/spring-projects/spring-ai/discussions/288 would allow configuring default and runtime seed `seed` values?

**Reply by Omkar-Shetkar** [reply]
Thanks @tzolov.
